### ISSUE TITLE
Hi, What if the prior probability of the noise label of the training set is not known?

### ISSUE THREAD

**Issue by ZunxiaoXu** [haochenglouis/cores#2]

**Comment by MuhammadSYahyaS** [comment]
I also have similar issue. In the paper itself, it is being said that
> The prior probability P(Y~|D~) is counted directly from the noisy dataset.

But, it is also being said that

> Beyond their results, we attempt to propose a theoretically sound approach addressing a general instance-based noise regime without knowing or estimating noise rates.

In the implementation, if we set `noise_prior = None`, then https://github.com/haochenglouis/cores/blob/cf0945e2d5eb4f38962913d51d6dff679099ad86/phase1.py#L211 will raise an error. I have tried:

- Initializing it with `noise_prior = np.ones(num_classes)`
- Avoiding the two lines of `noise_prior_cur` update process

and all on CIFAR-10 dataset, but my best result was the best test acc simply dropped from `69.28` (using the dataset's known noise prior) to `46.54`. Any lead about how to set the `noise_prior` on unknown prior probability of the dataset's noise? @haochenglouis
